.circleci/config.yml

```diff
@@ -215,28 +231,28 @@ jobs:
       name: go
     steps:
       - checkout
-      - run:
-          name: Install Terraform
-          command: |
-            curl -sLo terraform.zip https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
-            unzip terraform.zip
-            sudo cp terraform /usr/local/bin/
+      - setup-terraform
       - run: make terraform-fmt-check
 
 workflows:
-  version: 2
   build-test:
     jobs:
       - go-checks
       - terraform-checks
-      - compile_weekly_tests
-      - unit_integration_tests
-      - e2e_tests
+      - compile-weekly-tests
+      - unit-tests
```

Comment:
TODO: Remove

Reply:
Addressed in https://github.com/hashicorp/consul-terraform-sync/pull/869/commits/b6e8e0ee61a8137207278705c7c473ebb52d612e